### ISSUE TITLE
Browser (WASM) Accessibility for Screen Readers

### ISSUE THREAD

**Issue by stevemonaco** [AvaloniaUI/Avalonia#15453]
### Is your feature request related to a problem? Please describe.

WASM appears to lack support for accessibility for screen readers and possibly accessibility for WASM altogether.

### Describe the solution you'd like

Flutter in its canvas renderer approaches this by managing hidden, unrendered DOM elements with ARIA attributes.

### Additional context

This was posted on behalf of @asyncmeow
